spec/controllers/home_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe HomeController do
+  include Devise::Test::ControllerHelpers
+
+  let(:user) { create(:user) }
+  let(:org) { create(:team) }
+
+  before do
+    allow(controller).to receive(:current_user).and_return(user)
+    sign_in user
+  end
+
+  describe '#index' do
+    context 'when the user is signed in' do
+      it 'has a redirect response' do
+        get :index
+        expect(response.status).to eq(302)
+      end
+    end
+
+    context 'when the user is not signed in' do
+      before do
+        allow(controller).to receive(:current_user).and_return(nil)
```

Comment:
I wish there was an easy way to just override the `let(:user)` with `nil` but the `sign_out user` line below is making that tough